notificationworkerlambda/src/main/scala/com/gu/notifications/worker/tokens/TokenService.scala

```diff
@@ -12,9 +12,11 @@ import play.api.libs.json.{Format, Json}
 import scala.concurrent.ExecutionContextExecutor
 
 case class IndividualNotification(notification: Notification, token: String)
+case class BatchNotification(notification: Notification, token: List[String])
 
 case class ChunkedTokens(notification: Notification, tokens: List[String], range: ShardRange) {
   def toNotificationToSends: List[IndividualNotification] = tokens.map(IndividualNotification(notification, _))
+  def toBatchNotificationToSends: List[BatchNotification] = tokens.grouped(500).map(BatchNotification(notification, _)).toList
```

Comment:
This is the bit of logic that batches the tokens from the SQS message body into groups of 500. I think it was suggested to maybe group slightly less that firebase's maximum, so we could think about reducing this by 10%, for example?